docs/development.rst

```diff
@@ -44,40 +44,42 @@ independent parts which can be reviewed and merged individually.
 Automated Testing
 =================
 
-All pull requests and merges to 'master' branch are tested in `Travis <https://travis-ci.org/>`_
-based on our `.travis.yml file <https://github.com/pypa/pip/blob/master/.travis.yml>`_.
+All pull requests and merges to 'master' branch are tested using `Travis CI`_
+based on `our .travis.yml file`_.
 
-Usually, a link to your specific travis build appears in pull requests, but if not,
-you can find it on our `travis pull requests page <https://travis-ci.org/pypa/pip/pull_requests>`_
+Usually, a link to your specific Travis CI build appears in pull requests, but
+if not, you can find it on `our Travis CI Pull Requests page`_
 
-The only way to trigger Travis to run again for a pull request, is to submit another change to the pull branch.
+The only way to trigger Travis to run again for a pull request, is to submit
+another change to the pull branch.
 
-We also have Jenkins CI that runs regularly for certain python versions on windows and centos.
+We also have Jenkins CI that runs regularly for certain python versions on
+Windows and CentOS.
```

Comment:
Do we? I didn't know about that - we should include a link here. Honestly, I wonder if this is just out of date, and actually we don't have this...

Reply:
old. Will update.